### DISCUSSION TITLE
@mikro-orm/core/typings.js:54    Unexpected token ? Unexpected token ?

### DISCUSSION THREAD

**Discussion by adetorodev** [mikro-orm/mikro-orm#2952] [Q&A] [answered]
**Unexpected token ? error in core/typing.js line 54**

This is my first time trying Mikro-orm. I followed the documentation to set up Mikro-orm with PostgreSQL.

**index.ts**
```
import { MikroORM } from "@mikro-orm/core";
import mikroOrmConfig from "./mikro-orm.config";

const main = async () => {
    const orm = await MikroORM.init(mikroOrmConfig );
      console.log(orm.em);
}

main().catch((error) => {
    console.log(error)
});
...
```
**mikro-orm.config.ts file**
```
import { MikroORM, ReflectMetadataProvider } from "@mikro-orm/core";
import path from "path";
import { __prod__ } from "./constants";
import { Post } from "./entities/post";

export default {
  migrations: {
    path: path.join(__dirname, "./migrations"),
    glob: "!(*.d).{js,ts}",
  },
  allowGlobalContext: true,
  metadataProvider: ReflectMetadataProvider,
  entities: [Post],
  dbName: "tests",
  password: 'ezeka',
  type: "postgresql",
  debug: !__prod__,
} as Parameters<typeof MikroORM.init>[0];
```

**To Reproduce**
1. ...
2. ...
3. ...

**Expected behavior**
The ```console.log``` to log ```orm.em``` properties as described in the documentation.

**Versions**

| Dependency | Version |
| - | - |
| node | 10.9.0 |
| typescript | 4.6.2|
| mikro-orm | 5.1.0 |

**Comment by B4nan** [comment, marked as answer]
Upgrade your node version, and dont ignore warnings you get when you install things, as this is validated on npm level.

**Reply by adetorodev** [reply]
I have updated my node to 17. It is working perfectly well now. Thanks
